src/main/scala/io/citrine/lolo/bags/BaggerHelper.scala

```diff
@@ -1,71 +1,56 @@
 package io.citrine.lolo.bags
 
 import io.citrine.lolo.stats.{MathUtils, StatsUtils}
-import io.citrine.lolo.{Model, RegressionResult}
+import io.citrine.lolo.Model
 
 import scala.collection.parallel.immutable.ParSeq
 
 /**
-  * Helper class to subsume shared functionality of Bagger and MultiTaskBagger.
+  * Helper class to subsume shared functionality of [[RegressionBagger]] and [[MultiTaskBagger]].
   *
   * @param models collection of trained models
   * @param trainingData on which models were trained
   * @param Nib vector (over models) of vectors (over training data) of the number of repeats in each model's bag
   * @param useJackknife whether to use jackknife for uncertainty quantification
   * @param uncertaintyCalibration whether to apply empirical uncertainty calibration
   */
-protected case class BaggerHelper(
-    models: ParSeq[Model[Any]],
-    trainingData: Seq[(Vector[Any], Any)],
+protected[bags] case class BaggerHelper(
```

Comment:
The only active path through this helper class was on regression tasks, where it was used to calculate OOB errors and the uncertainty rescale. After breaking up the regression and classification baggers, I was able to make this helper type-specific for real-valued data.